src/main/java/org/chocosolver/solver/search/measure/MeasuresRecorder.java

```diff
@@ -98,8 +98,8 @@ public void reset() {
         state = SearchState.NEW;
         objectiveOptimal = false;
         solutionCount = 0;
-        timeCount = 0;
         stopStopwatch();
+        timeCount = 0;
```

Comment:
It could be good to add a comment that it is essential that the assignment `timeCount = 0` is after the call to `stopStopwatch()`.